main.js

```diff
@@ -1379,10 +1379,13 @@ function connect() {
         reconnection: true,
         rejectUnauthorized: !adapter.config.allowSelfSignedCertificate,
         reconnectionDelay:    5000,
-        timeout:              5000,
+        timeout:              adapter.config.connectionTimeout,
```

Comment:
Maybe also a fallback for an undefined value here because of backward compatibility? like
adapter.config.connectionTimeout || 5000